### PR TITLE
LPS-191864 Adding test for the sort in the views tab

### PR DESCRIPTION
**Pull request by JerryAdrianoo**
**Description:**
Adding test for the sort in the views tab.

**Jira Ticket:**
https://liferay.atlassian.net/browse/LPS-191864

### CODE REVIEW

**Comment by JerryAdrianoo** [comment]
ci:test:frontend-data-set

**Comment by JerryAdrianoo** [comment]
@CarlosBrichete these failures are not related!!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetView.testcase [hunk 1]
```diff
@@ -623,4 +623,49 @@ definition {
 		}
 	}
 
+	@description = "LPS-189969 - Verify if when the user changes the name of one view, the table will be sorted by the names"
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetView.testcase:626]: Shouldn't it be "sorted by the modified date"?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetView.testcase [hunk 2]
```diff
@@ -623,4 +623,49 @@ definition {
...
+	@priority = 3
+	test SortViewsByName {
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetView.testcase:628]: The purpose of this test is to check that the views are ordered by modified date, from newest to oldest. The name of the test seems like we are testing something different. Please, change the name of the test to something like "ViewsSortedByModifiedDate"

**Review by CarlosBrichete — CHANGES_REQUESTED** [review]

**Comment by JerryAdrianoo** [comment]
ci:test:sf

**Comment by JerryAdrianoo** [comment]
@CarlosBrichete Can I forward it?

**Comment by CarlosBrichete** [comment]
yes, forward it! Thanks :)

**Comment by JerryAdrianoo** [comment]
ci:forward

**Pull request closed without merging**